### ISSUE TITLE
Empty space in popup activates scrolling on vertical stack card 

### ISSUE THREAD

**Issue by vcntt** [Clooos/Bubble-Card#234]
Whenever I open a popup card and scroll on empty space the vertical stack card beneath it will activate. 

https://github.com/Clooos/Bubble-Card/assets/34486569/5da1462b-f858-4c5b-921a-f4cb9b7da4c7

**Comment by Paxxil** [comment]
try this
```
  styles: |
    #root {
      overscroll-behavior: contain !important;
    }    
```

**Comment by Clooos** [comment]
This is now fixed in the new v1.6.0 beta!

https://github.com/Clooos/Bubble-Card/releases/tag/v1.6.0-beta.1
